components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/internal/cache/ApplicationBasicInfoCacheEntry.java

```diff
@@ -21,12 +21,12 @@
 /**
  * Cache entry for basic information for the application.
  */
-public class ApplicationBasicInfoResourceIdCacheEntry extends CacheEntry {
+public class ApplicationBasicInfoCacheEntry extends CacheEntry {
```

Comment:
Changing this class name as it is inside the internal package of the component. So no external party will be affected.